designs/developer/add-45/sections/pipelines.md

```diff
@@ -0,0 +1,80 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Pipelines section in Add Flows (added to 4.3)
+The Pipelines section allows the user to enable a pipeline template for their application, component or service.
+* A Pipelines section appears in +Add flows (for Import from Git,  Developer Catalog, and From Dockerfile) when the Pipelines operator is installed.
+* When the Pipelines operator is not installed, this section is not displayed
+* The Pipelines section has a “Dev Preview” badge
+
+## Import from Git
+### No Builder Image Selected
+* When a builder image has not been selected, the Pipelines option shall display an Info alert that says “Select a builder image to see if there is a pipeline template available for this runtime.”
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_No_Img.png)
```

Comment:
if you keep the relative path here, you'll have to add the image in the designs/developer/add-45/sections/img directory, or you can point back to the 4.4 image.

Reply:
@serenamarie125 I think most of the images for this section will change. Just some of the intro text will stay the same. But we should definitely repurpose any images instead of recreating where possible. I'm going to work with Laura on this one today.

Reply:
agreed - For now, you could point to 44 if you wanted, so the images will be available, and then laura could do her PR afterwards with the new updates.  Whatever is easiest!

Reply:
OK.